corehq/privileges.py

```diff
@@ -79,6 +79,8 @@
 
 APP_USER_PROFILES = 'app_user_profiles'
 
+DEFAULT_EXPORT_SETTINGS = 'default_export_settinngs'
```

Comment:
Fixed this typo [here](https://github.com/dimagi/commcare-hq/pull/29366) and properly updated staging since this migration was already applied.